src/notify-middleware.js

```diff
@@ -2,17 +2,19 @@ import * as actions from "./login/components/Notifications/Notifications_actions
 
 const notifyAndDispatch = () => next => action => {
   if (action.type.endsWith("SUCCESS") || action.type.endsWith("FAIL")) {
-    if (action.error && action.payload) {
-      console.log("this is action.payload", action.payload);
-      console.log("this is action.error", action.error);
+    if(action.type.startsWith("GET_LETTER")){
```

Comment:
Can this be solved in the reducer?
https://github.com/SUNET/eduid-front/blob/eunju-fix-notify-error/src/reducers/LetterProofing.js#L42